GSoC-2025/pranav_kapoor.md

```diff
@@ -0,0 +1,80 @@
+
+My project aims to enhance the utility of the BrainGlobe ecosystem by updating the atlas packaging scripts so they output data in the OpenMINDS SANDS format. This upgrade is critical for researchers who need consistent, comprehensive data to integrate findings across studies and species. By delivering a modular, well-documented solution complete with rigorous testing, I will enable both seasoned neuroscientists and new entrants in the field, making advanced neuroinformatics more accessible to all.
+
+### Implementation timeline
+
+1. **Minimal Deliverables:**
+   - Modify existing atlas packaging code to output atlas data conforming to the OpenMINDS SANDS standard.
+   - Adapt at least one packaging script (e.g., the Allen Human atlas script) to incorporate this new functionality.
+   - Develop comprehensive tests covering all new and modified features.
+   - Update the documentation to guide users on how to generate and utilize OpenMINDS SANDS‑compliant atlases.
+2. **Stretch Goals (if time permits):**
+   - Extend support for additional species or imaging modalities.
+   - Integrate schema validation using tools like `jsonschema` to ensure full compliance with OpenMINDS SANDS.
+   - Create a conversion utility for existing atlas packages to the new standard.
+3. **Time Commitment:**  
+   I will be devoting 30–35 hours per week towards this GSoC project.
+
+| **Week** | **Tasks & Deliverables**                                                                                                                                                                                                                                         | **Hours** |
+|----------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------|
+| 1–2      | **Community Bonding & Research:**<br>- Immerse myself in the BrainGlobe Atlas API, current packaging scripts, and OpenMINDS SANDS documentation.<br>- Engage with mentors to clarify requirements and set up a focused development environment.      | 30        |
+| 3–4      | **Design & Planning:**<br>- Map the existing atlas data to the OpenMINDS SANDS schema.<br>- Design a modular function (e.g., `wrapup_atlas_to_openminds`) and create clear wireframes for the conversion workflow.                                      | 30        |
+| 5–6      | **Core Development – Part 1:**<br>- Develop the new wrap‑up function in a separate module (e.g., `openminds_wrapup.py`).<br>- Adapt the Allen Human atlas packaging script to integrate this function and produce the new output.                         | 30        |
```

Comment:
Just because the human atlas is a low priority atlas, I would change this to the Allen mouse atlas.